openpype/modules/kitsu/utils/update_op_with_zou.py

```diff
@@ -346,6 +347,18 @@ def sync_project_from_kitsu(dbcon: AvalonMongoDB, project: dict):
     if not project:
         project = gazu.project.get_project_by_name(project["name"])
 
+    # Get all statuses for projects from Kitsu
+    all_status = gazu.project.all_project_status()
+    for status in all_status:
+        if project['project_status_id'] == status['id']:
+            project['project_status_name'] = status['name']
+
```

Comment:
Would this benefit from a comment?
```suggestion

    # Do not sync closed kitsu project that is not found in openpype
```